docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,700 @@
+
+```js
+UID2Manager.init(
+  context = this,
+  serverUrl = "https://operator-integ.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+// Must be set before UID2Manager.shared is accessed
+UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://operator-integ.uidapi.com")!
+)
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+Bear in mind the following differences between environments:
+- Tokens from the UID2 integration environment are not valid for passing to the bid stream.
+- You'll have a different set of Subscription ID and public key values for each environment (integ and prod). Be sure to use the correct values for each environment.
+:::
+
+### Optional: Reduce Latency by Setting the API Base URL for the Production Environment
+
+<ReduceLatency />
+
+To specify a different UID2 server, you can change it in the `init` call, as shown in the following examples.
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+ 
+```js
+UID2Manager.init(
+  context = this,
+  serverUrl = " https://global.prod.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+// Must be set before UID2Manager.shared is accessed
+UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://global.prod.uidapi.com")!
+)
+// or use a named environment
+UID2Settings.shared.environment = .sydney
+```
+
+</TabItem>
+</Tabs>
+
+## Configure the UID2 Mobile SDK
+
+UID2 provides the publisher with the following values, which are needed for generating the UID2 token on the client side:
+
+- Subscription ID
+- Public key
+
+You'll have one set of these values for your testing environment, and a separate set for your production environment.
+
+To configure the SDK, you must pass in the Subscription ID and public key that you received during account setup, as well as the user’s hashed or unhashed directly identifying information (<Link href="../ref-info/glossary-uid#gl-dii">DII</Link>) (email address or phone number), into the following method call:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().generateIdentity()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.generateIdentity()
+```
+
+</TabItem>
+</Tabs>
+
+Once it's configured, the UID2 mobile SDK does the following:
+
+- Generates a UID2 token for the user
+- Stores the token locally on the user’s device
+- Automatically refreshes the token as required while your app is open
+
+:::tip
+You can pass the user’s <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> to the UID2 mobile SDK either hashed or unhashed. If you pass the DII unhashed, the SDK hashes it for you. If you want to pass the DII to the SDK already hashed, you must normalize it before hashing. For details, see [Normalization and Encoding](../getting-started/gs-normalization-encoding.md).
+:::
+
+## Format Examples for DII
```

Comment:
i think this should be a subsection under `Configure the UID2 Mobile SDK` 

Not strong opinion tho if this causes other formatting issue....

Reply:
Holding off on this, for now anyway.

Reply:
@sunnywu do you want this mod in the JS doc, publisher-client-side, as well?